Web/di.yml

```diff
@@ -42,3 +43,4 @@ services:
     - openvk\Web\Models\Repositories\Topics
     - openvk\Web\Models\Repositories\Applications
     - openvk\Web\Models\Repositories\ContentSearchRepository
+    - openvk\Web\Models\Repositories\Blacklists
```

Comment:
https://ru.wikipedia.org/wiki/%D0%9F%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4_%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%B8